server/fixtures/category.js

```diff
@@ -0,0 +1,8 @@
+export const newCategory = {
+  name: 'default'
+};
+
+export const newCategoryData = args => ({
```

Comment:
I still don't see the need for this function with an object with only one parameter